src/bpmn-visu-js/mx-graph-visu.ts

```diff
@@ -0,0 +1,19 @@
+import { mxgraphFactory } from './../../node_modules/ts-mxgraph/index';
```

Comment:
:question: shouldn't we able to only put ts-mxgraph thanks to `@rollup/plugin-node-resolve`?

Reply:
`@rollup/plugin-node-resolve` is only for the final bundle, so won't apply here